src/main.c

```diff
@@ -344,6 +344,8 @@ main(int argc, char** argv)
    struct main_configuration* config = NULL;
    int ret;
    int c;
+   char os[64];
```

Comment:
should we use a `#define` for this repeasted `64`? and `malloc` it as a `char *` instead of an array?